clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -162,7 +162,7 @@
                                         CommonClientConfigs.SOCKET_CONNECTION_SETUP_TIMEOUT_MS_DOC)
                                 .define(SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS_CONFIG,
                                         Type.LONG,
-                                        CommonClientConfigs.DEFAULT_SOCKET_CONNECTION_SETUP_TIMEOUT_MAX_MS,
```

Comment:
I think the producer and consumer have the same problem. Maybe we can just change the constant?

Reply:
seems reasonable.